modules/redis/main.tf

```diff
@@ -11,8 +11,4 @@ resource "azurerm_redis_cache" "default" {
   redis_configuration {
   }
 
-  tags = {
```

Comment:
should we set tags here?

Reply:
Yes, but, cleaning up tagging in this repo is a separate PR.